dashboards-observability/.cypress/utils/app_constants.js

```diff
@@ -47,11 +47,11 @@ export const changeTimeTo24 = (timeUnit) => {
   cy.get('[aria-label="Time unit"]').select(timeUnit);
   cy.get('.euiButton').contains('Apply').click();
   cy.wait(delay);
-  cy.get('.euiButton').contains('Refresh').click();
+  cy.get('[data-test-subj="superDatePickerApplyTimeButton"]').click();
 };
 
-export const expectMessageOnHover = (message) => {
-  cy.get('.euiToolTipAnchor').contains('Create').click({ force: true });
+export const expectMessageOnHover = (button, message) => {
+  cy.get(`[data-test-subj="${button}"]`).click({ force: true });
```

Comment:
Did we make all cypress changes in functional tests repo?

Reply:
probably didn't since I saw there's one open PR from @eugenesk24 in FTR. But I don't think keep FTR sync with our cypress tests is our strategy moving forward.